src/transformers/pipelines/__init__.py

```diff
@@ -611,6 +611,21 @@ def pipeline(
                 feature_extractor, revision=revision, _from_pipeline=task, **model_kwargs
             )
 
+            if (
+                feature_extractor._processor_class
+                and feature_extractor._processor_class.endswith("WithLM")
+                and isinstance(model_name, str)
+            ):
+                try:
+                    from pyctcdecode import BeamSearchDecoderCTC
+
+                    decoder = BeamSearchDecoderCTC.load_from_hf_hub(model_name)
```

Comment:
```suggestion
            language_model_filenames os.path.join(BeamSearchDecoderCTC._LANGUAGE_MODEL_SERIALIZED_DIRECTORY, "*")
            alphabet_filename = BeamSearchDecoderCTC._ALPHABET_SERIALIZED_FILENAME
            allow_regex = [language_model_filenames, alphabet_filename]
            
            decoder = BeamSearchDecoderCTC.load_from_hf_hub(
                pretrained_model_name_or_path, allow_regex=allow_regex, **kwargs
            )
```

Reply:
This will prevent the load command to unnecessarily download model files, etc...